freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/user/handler/FreeIpaClientEventHandler.java

```diff
@@ -0,0 +1,8 @@
+package com.sequenceiq.freeipa.flow.freeipa.user.handler;
+
+import reactor.bus.Event;
+import reactor.fn.Consumer;
+
+public interface FreeIpaClientEventHandler<T> extends Consumer<Event<T>> {
```

Comment:
I recommend to use `com.sequenceiq.flow.reactor.api.handler.EventHandler` as the parent class of the new reactor event handlers instead of this class.

Reply:
done